docs/docs/dev_docs/getting_started/sandbox.md

```diff
@@ -290,30 +312,39 @@ We can see that each account has the expected balance of tokens.
 
 ## Creating and submitting transactions
 
-Now lets transfer some funds from Alice to Bob by calling the `transfer` function on the contract. This function takes 3 arguments:
+Now lets transfer some funds from Alice to Bob by calling the `transfer` function on the contract. This function takes 4 arguments:
 
-1. The quantity of tokens to transfer.
-2. The sender.
-3. The recipient.
+1. The sender.
+2. The recipient.
+3. The quantity of tokens to be transferred.
+4. The nonce for the AuthWit, or 0 if msg.sender equal sender.
```

Comment:
Remember to link to AuthWit docs if there are any, or use the full Auth Witness, this is a tutorial and i have no context to what this is yet

Reply:
Expanded it for authentication witness and added a link.